lib/HTFeed/BackupExpiration.pm

```diff
@@ -43,18 +50,27 @@ sub run {
     HAVING COUNT(*) > 1
 SQL
 
+  my $versions_sth = get_dbh()->prepare(<<'SQL');
```

Comment:
Small optimization, but I noticed we were re-preparing the statements for every object, and it didn't seem like we needed to do that. @moseshll I did want to check there wasn't any particular reason for that though.

Reply:
No, my experience with the MySQL DBI in Perl has been to sidestep statement handles, so I don't tend to use them in particularly elegant ways.